orte/tools/orterun/orterun.1in

```diff
@@ -13,10 +13,12 @@
 .SH NAME
 .
 orterun, mpirun, mpiexec \- Execute serial and parallel jobs in Open MPI.
+oshrun, shmemrun \- Execute serial and parallel jobs in Open SHMEM.
```

Comment:
how come. what do I miss: the patch looks:

``` shell
+oshrun, shmemrun \- Execute serial and parallel jobs in Open SHMEM.
```
it adds new line with synonyms under existing line, w/o changing it?